src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -22,7 +22,7 @@ const TableHeader: React.FC<TableHeaderProps> = ({
 }) => {
   const { user, profile } = useUser();
 
-  const experiences = useSelector(experiencesSelector);
+  const experience = useSelector(experiencesSelector);
```

Comment:
Why the change to experience (singular) here? The selector looks like it still refers to experiences (plural)?